node/node.go

```diff
@@ -160,7 +160,8 @@ func (n *Node) Start(ctx context.Context) error {
 			coreDB.Blockstore(),
 			coreDB.Encstore(),
 			coreDB.Events(),
-			NewNetACP(acp, coreDB),
+			acp,
+			coreDB,
```

Comment:
suggestion: At this point do we just want to remove the `coreDB.XYZ` args above and just consume the one `DB` arg (`coreDB` here)

Reply:
Fair point. I'll look into it.

Reply:
Made the change 👍